baseapp-profiles/baseapp_profiles/models.py

```diff
@@ -155,6 +156,11 @@ class Meta:
         abstract = True
 
 
+@pghistory.track(
+    pghistory.InsertEvent(),
+    pghistory.UpdateEvent(),
+    pghistory.DeleteEvent(),
+)
```

Comment:
its better to remove `pghistory.track` from here and instead add it in `apps.social.profiles.models.Profile`. I had some migrations going crazy with `pghistory.track` added to a default swappable model. Maybe you will need to make a profiles app in your testproject as well

Reply:
Thanks a lot :pray: . I had no idea.

I did what you suggested :)